programs/protocol_event/src/instructions/create_grouping.rs

```diff
@@ -0,0 +1,28 @@
+use crate::context::{CreateCategory, CreateEventGroup};
+use anchor_lang::prelude::*;
+
+pub fn create_category(ctx: Context<CreateCategory>, code: String, name: String) -> Result<()> {
```

Comment:
Do we want to be strict about leaving `Context` and `Account` types behind in `lib.rs` and only having method signatures we can write unit tests for in `instructions/*.rs`?

Reply:
Yeah that is a good point, I have actually done that in my other PRs but will make sure these base PRs follow suit